vector/src/main/java/im/vector/adapters/VectorMediaViewerAdapter.java

```diff
@@ -368,7 +369,9 @@ public void onSuccess(File mediaFile) {
                                                     @Override
                                                     public void run() {
                                                         Uri mediaUri = Uri.parse(newHighResUri);
-                                                        Glide.with(imageView).load(mediaUri).into((ImageView) imageView);
+                                                        Glide.with(imageView)
```

Comment:
Could be interesting to keep reference to glideRequests (=Glide.with(context); 
will allow to call glideRequest.clear() when media preview is dismissed to clear memory

Reply:
Glide memory is cleared when Activity is destroyed (you made the check ;-))